docs/products-solutions.asciidoc

```diff
@@ -3,8 +3,11 @@
 
 The following Elastic products support ECS out of the box, as of version 7.0:
 
-* {beats}
-* APM
+* {beats-ref}/beats-reference.html[{beats}]
+* {apm-get-started-ref}/overview.html[APM]
+* {siem-guide}[Elastic SIEM app]
+* https://www.elastic.co/products/endpoint-security[Elastic Endpoint Security 
+Server]
 * Infrastructure UI and Logs UI
```

Comment:
Let's take this opportunity to rephrase this bullet point to as well to "Logs monitoring UI", and link it to the docs: http://ecs_622.docs-preview.app.elstc.co/guide/en/logs/guide/current/logs-app-overview.html

Reply:
Updated, but I'll change the the URL when 7.5 is released and becomes the current branch .

Reply:
Oh I didn't even realize this was a brand new page :-)